packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -41,20 +41,26 @@ export const configSchema = {
     _elements: {
       _type: Type.UUID,
     },
-    _description: 'The identifier types to be display on all patient search result page',
+    _description:
+      'A list of identifier types to be displayed in the patient search results as banner tags. If no defaultIdentifierTypes are provided, the defaultIdentifier will be displayed.',
+    // These values correspond to the Patient Clinic Number and National Unique Patient Identifier (NUPI) identifier types respectively
     _default: ['b4d66522-11fc-45c7-83e3-39a1af21ae0d', 'f85081e2-b4be-4e48-b3a4-7994b69bb101'],
   },
   defaultIdentifier: {
     _type: Type.String,
-    _description: 'Identifier to be shown in the event defaultIdentifierTypes does is empty',
+    _description: 'Identifer type to be displayed when no defaultIdentifierTypes are provided. Default is OpenMRS ID.',
     _default: 'OpenMRS ID',
   },
 };
 
 export type PatientSearchConfig = {
-  search: { patientSearchResult: string; showRecentlySearchedPatients: string; disableTabletSearchOnKeyUp: boolean };
-  includeDead: boolean;
+  search: {
+    disableTabletSearchOnKeyUp: boolean;
```

Comment:
Is disableTabletSearchOnKeyUp used only for tablets? Or can it be also used in other cases?

Reply:
It's in all other cases, but this is just adding the existing configuration item (see line 15 above) to the TS type for the schema.